materialrangebar/src/main/java/com/appyvet/materialrangebar/RangeBar.java

```diff
@@ -206,12 +212,22 @@
 
     private int mActiveConnectingLineColor;
 
+    private ArrayList<Integer> mActiveConnectingLineColors = new ArrayList<>();
+
     private int mActiveBarColor;
 
-    private int mActiveTickColor;
+    private int mActivemTickDefaultColor;
```

Comment:
Typo here:
`mActivemTickDefaultColor` should be renamed to `mActiveTickDefaultColor`

Reply:
Changed